contrib/python/README.md

```diff
@@ -1,14 +1,25 @@
 Python contributions for ASPECT 
 ==============================
 
-This folder contains some useful routines to read files with Python,
-using the numpy and pandas packages.
+This folder contains python function that ASPECT users can find useful.
+
+Python packages needed:
+- numpy
+- io 
+- os
+- subprocess
+- sys 
+- pandas
+- xmltodict
+- requests
+- matplotlib
+- pyvista
```

Comment:
Please add an environment.yml file like for the documentation (see /doc/sphinx/environment.yml). This way one can set up the conda environment as `conda env create -f environment.yml`.

Reply:
Done